packages/dev/codemods/src/s1-to-s2/src/iconMap.ts

```diff
@@ -56,6 +60,10 @@ export const iconMap = new Map([
     'AnchorSelect',
     'DirectSelect'
   ],
+  [
+    'App',
+    'App'
+  ],
   [
     'Apps',
```

Comment:
I think this map is incorrect? we have both of these, Apps and AppsAll

Reply:
@reidbarber any idea?

Reply:
Looked it up and S1 Apps -> S2 AppsAll is the migration, while S2 Apps is a new icon. So this looks to be intentional.